sympy/ntheory/modular.py

```diff
@@ -253,3 +255,37 @@ def combine(c1, c2):
         if symmetric:
             return symmetric_residue(n, m), m
         return n, m
+
+
+def crt_cartesian(rem, mod):
+    """
+    Return a list of values whose elements give the remainders
+    in the Cartesian product of the supplied remainders when divided
+    by each of the supplied moduli. If the moduli are not co-prime,
+    you will receive a ValueError.
+
```

Comment:
IMO, the following section should be included in the docstring,
```python
Parameters
=========

parameter_name: type_of_parameter
    Explanation for the parameter, what value it defaults, etc.
```

Reply:
This section is missing not only in ```crt_cartesian``` but also in ```crt```. I will include them in both